src/components/asignarTarea.js

```diff
@@ -5,49 +5,38 @@ import { UsuarioService } from '../services/usuarioService'
 import { Tarea } from '../domain/tarea'
 import { PropTypes } from 'prop-types'
 
-const tareaService = new TareaService()
-const usuarioService = new UsuarioService()
 
 export default class AsignarTareaComponent extends Component {
 
-  constructor(props) {
-    super(props)
-    this.state = {
-      usuarios: [],
-      tarea: new Tarea()
-    }
-    this.initialize()
+  state = {
+    usuarios: [],
+    tarea: new Tarea()
   }
 
-  async initialize() {
+  async componentDidMount() {
     try {
-      const response = await usuarioService.allInstances()
-      const usuarios = await response.json()
-      const tarea = await tareaService.getTareaById(this.props.match.params.id)
+      const usuarios = await UsuarioService.allInstances()
+      const tarea = await TareaService.getTareaById(this.props.match.params.id)
       this.setState({
-          usuarios: usuarios,
-          tarea: tarea
+        usuarios: usuarios,
+        tarea: tarea
       })
     } catch (e) {
       this.generarError(e)
     }
   }
 
-  snackbarOpen() {
-    return !!this.state.errorMessage
-  }
-
-  async asignarTarea() {
+  asignarTarea = async () => {
     try {
       this.state.tarea.validarAsignacion()
-      await tareaService.actualizarTarea(this.state.tarea)
+      await TareaService.actualizarTarea(this.state.tarea)
       this.volver()
     } catch (e) {
       this.generarError(e)
     }
   }
 
-  cambiarEstado(closureChange) {
+  cambiarEstado = (closureChange) => {
```

Comment:
no se qué ventaja tiene definir una lambda en lugar de un método que ya conocen de Angular, hay algún cambio significativo?

Reply:
Si las escribimos como metodos normales, tenemos que bindearles a mano el `this` en el constructor: 
```this.cambiarEstado = this.cambiarEstado.bind(this)```
Y por que esto funcionaba antes  ? por que dentro del jsx se hacia algo de este estilo por ejemplo :
```js
<ComponenteCrazy onClick={ (event) => this.onClick(event) } />
```
Por lo tanto la lambda que estabas generando a mano hacia que el this sea el del componente

Reply:
buenísimo, no se si está en el README aclarado pero es un excelente punto para transmitir

Reply:
Dale ahora lo agrego :D